src/Magnum/Platform/GlfwApplication.h

```diff
@@ -0,0 +1,649 @@
+         *
+         * @see @ref setFlags()
+         */
+        #ifndef DOXYGEN_GENERATING_OUTPUT
+        typedef Containers::EnumSet<Flag, GLFW_CONTEXT_NO_ERROR | GLFW_CONTEXT_RELEASE_BEHAVIOR> Flags;
+        #else
+        typedef Containers::EnumSet<Flag> Flags;
+        #endif
+
+        /*implicit*/ Configuration();
+        ~Configuration();
+
+        /** @brief Window title */
+        std::string title() const { return _title; }
+
+        /**
+         * @brief Set window title
+         * @return Reference to self (for method chaining)
+         *
+         * Default is `"Magnum GLFW Application"`.
+         */
+        Configuration& setTitle(std::string title) {
+            _title = std::move(title);
+            return *this;
+        }
+
+        /** @brief Window size */
+        Vector2i size() const { return _size; }
+
+        /**
+         * @brief Set window size
+         * @return Reference to self (for method chaining)
+         *
+         * Default is `{800, 600}`.
+         */
+        Configuration& setSize(const Vector2i& size) {
+            _size = size;
+            return *this;
+        }
+
+        /** @brief Context flags */
+        Flags flags() const { return _flags; }
+
+        /**
+         * @brief Set context flags
+         * @return Reference to self (for method chaining)
+         *
+         * Default is no flag.
+         */
+        Configuration& setFlags(Flags flags) {
+            _flags = flags;
+            return *this;
+        }
+
+        /** @brief Context version */
+        Version version() const { return _version; }
+
+        /**
+         * @brief Set context version
+         *
+         * If requesting version greater or equal to OpenGL 3.1, core profile
+         * is used. The created context will then have any version which is
+         * backwards-compatible with requested one. Default is
+         * @ref Version::None, i.e. any provided version is used.
+         */
+        Configuration& setVersion(Version version) {
+            _version = version;
+            return *this;
+        }
+
+        /** @brief Sample count */
+        Int sampleCount() const { return _sampleCount; }
+
+        /**
+         * @brief Set sample count
+         * @return Reference to self (for method chaining)
+         *
+         * Default is `0`, thus no multisampling. The actual sample count is
+         * ignored, GLFW either enables it or disables. See also
+         * @ref Renderer::Feature::Multisampling.
+         */
+        Configuration& setSampleCount(Int count) {
+            _sampleCount = count;
+            return *this;
+        }
+
+        /** @brief sRGB-capable default framebuffer */
+        bool isSRGBCapable() const {
+            return _srgbCapable;
+        }
+
+        /**
+         * @brief Set sRGB-capable default framebuffer
+         * @return Reference to self (for method chaining)
+         */
+        Configuration& setSRGBCapable(bool enabled) {
```

Comment:
Hmm. I also wondered about this. Currently the `Flag` enum maps directly to whatever platform-specific bit flags there are (so SDL has its own, GLX, WGL, CGL, EGL also have its own) and unfortunately "sRGB capable" is not in those bit flags on neither of those and one has to use special value/function for it. So it made more sense to me to put it separately instead of having a big switch for the `Flag` values to decide whether they belong into context flags or somewhere else.

Let's do it consistently with the rest first, we can change later.